docs/doc/02-enterprise/10-enterprise-features.md

```diff
@@ -6,6 +6,7 @@ This page provides an updated list of available enterprise features. To access t
 
 | Feature                                                                                        	 | Description                                                                                                                                                                                                                                                                                                                                                                                 	 |
 |--------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Stream](/sql/sql-commands/ddl/stream) | Efficient data change management with stream:<br/>-Instantly capture both insertions and deletions in real-time.<br/>-Seamlessly leverage streams for direct querying and analysis, ensuring swift insights.|
```

Comment:
Currently only append.

For efficiency reasons, the APPEND_ONLY behavior of databend will merge (squash) deletions between two offsets. Rows that are deleted after append (and possibly subsequent update) between two offsets will not be recorded in the final changeset.

Reply:
A Databend stream currently supports **Append-only** mode, allowing it to capture data insertions. This encompasses potential data deletions or updates for the recently added rows. However, the stream consistently reflects the final changes to the table since its creation. For example, if a row is added and subsequently updated with new values, the stream will accurately describe the changes based on the updated values.